src/main/drivers/bus_quadspi.h

```diff
@@ -40,7 +40,7 @@
 
 #ifdef USE_QUADSPI
 
-#if !defined(STM32H7)
+#if !defined(STM32F7) || !defined(STM32H7)
```

Comment:
This will now fail for all targets that do not define `STM32F7` **and** `STM32H7`.

Should be

```
#if !(defined(STM32F7) || defined(STM32H7))
```

Reply:
Fixed it.